demisto_sdk/commands/content_graph/strict_objects/common.py

```diff
@@ -129,3 +130,79 @@ def create_dynamic_model(
     default=None,
     include_without_suffix=True,
 )
+KEY_DYNAMIC_MODEL = create_dynamic_model(
+    field_name="key",
+    type_=Optional[str],
+    default=None,
+)
+VALUE_DYNAMIC_MODEL = create_dynamic_model(
+    field_name="value",
+    type_=Optional[Any],
+    default=None,
+)
+PLAYBOOK_INPUT_QUERY_DYNAMIC_MODEL = create_dynamic_model(
+    field_name="playbookInputQuery",
+    type_=Optional[Any],
+    default=None,
+)
+FORM_DYNAMIC_MODEL = create_dynamic_model(
+    field_name="form",
+    type_=Optional[Dict],
+    default=None,
+    include_without_suffix=True,
+)
+MESSAGE_DYNAMIC_MODEL = create_dynamic_model(
+    field_name="message",
+    type_=Optional[Dict],
+    default=None,
+    include_without_suffix=True,
+)
+SCRIPT_ARGUMENTS_LOWER_CASE_DYNAMIC_MODEL = create_dynamic_model(
+    field_name="scriptarguments",
+    type_=Optional[Dict],
+    default=None,
+    include_without_suffix=True,
+)
+SCRIPT_ARGUMENTS_UPPER_CASE_DYNAMIC_MODEL = create_dynamic_model(
+    field_name="scriptArguments",
+    type_=Optional[Dict],
+    default=None,
+    include_without_suffix=True,
+)
+SCRIPT_ID_DYNAMIC_MODEL = create_dynamic_model(
+    field_name="scriptId",
+    type_=Optional[str],
+    default=None,
+    include_without_suffix=True,
+)
+IS_CONTEXT_DYNAMIC_MODEL = create_dynamic_model(
+    field_name="iscontext",
+    type_=Optional[bool],
+    default=None,
+    include_without_suffix=True,
+)
+
+
+class _LeftOrRight(BaseStrictModel):
+    value: Any
+
+
+LeftOrRight = create_model(
+    model_name="LeftOrRight",
+    base_models=(_LeftOrRight, VALUE_DYNAMIC_MODEL, IS_CONTEXT_DYNAMIC_MODEL),
+)
```

Comment:
a `value` defined both in line 187 and in the dynamic model? We can just 

```suggestion
LeftOrRight = create_model(
    model_name="LeftOrRight",
    base_models=(VALUE_DYNAMIC_MODEL, IS_CONTEXT_DYNAMIC_MODEL),
)
```

Reply:
ah i see, the raw `value` isn't in the imported one, ok